lib/publishable/active_record.rb

```diff
@@ -48,6 +48,14 @@ def publishable(options = {})
             end.order{[publication.publication_group.number.asc, publication.version.desc]}
           }
 
+          # retrieve all versions for number, regardless of published status
+          scope :versions_for_number, ->(number) {
```

Comment:
Maybe this should be a class method? Seems weird for it to be a scope if it's not chainable.